packages/cli/fern-definition/validator/src/rules/valid-path-parameters-configuration/valid-path-parameters-configuration.ts

```diff
@@ -0,0 +1,26 @@
+import { Rule } from "../../Rule";
+
+export const ValidPathParametersConfigurationRule: Rule = {
+    name: "valid-path-parameters-configuration",
+    create: () => {
+        return {
+            definitionFile: {
+                httpEndpoint: ({ endpoint }) => {
+                    if (
+                        endpoint["path-parameters"] != null &&
+                        typeof endpoint.request !== "string" &&
+                        endpoint?.request?.["path-parameters"] != null
+                    ) {
+                        return [
+                            {
+                                severity: "error",
+                                message: "Endpoint has path-parameters defined in both endpoint and request."
```

Comment:
nit: would update this error message to `path-parameters cannot be defined in both endpoint and request`